.factory/automation.yml

```diff
@@ -143,4 +143,4 @@ release:
       command: |
         export DEPLOY_CRATE_TOKEN=$REPO_CRATES_TOKEN
         bazel run --define version=$(cat VERSION) //grpc/rust:deploy_crate -- release
-      dependencies: [build, build-dependency]
+      dependencies: [deploy-github]
```

Comment:
`[deploy-github]` is the correct dependency list. This was not previously caught in CI because of:
- https://github.com/vaticle/factory/issues/1904